src/PodCreateModal.jsx

```diff
@@ -92,16 +96,22 @@ export const PodCreateModal = ({ user, selinuxAvailable, systemServiceAvailable,
     const defaultBody = (
         <Form>
             {dialogError && <ErrorNotification errorMessage={dialogError} errorDetail={dialogErrorDetail} />}
-            <FormGroup fieldId='create-pod-dialog-name' label={_("Name")} className="ct-m-horizontal"
-                    validated={nameError ? "error" : "default"}
-                    helperTextInvalid={nameError}>
+            <FormGroup fieldId='create-pod-dialog-name' label={_("Name")} className="ct-m-horizontal">
                 <TextInput id='create-pod-dialog-name'
                        className="pod-name"
                        placeholder={_("Pod name")}
                        value={podName}
-                       validated={nameError ? "error" : "default"}
                        aria-label={nameError}
                        onChange={value => onValueChanged('podName', value)} />
+                {nameError && (
+                    <FormHelperText>
+                        <HelperText>
+                            <HelperTextItem icon={<ExclamationCircleIcon />} variant="error">
+                                {nameError}
+                            </HelperTextItem>
+                        </HelperText>
+                    </FormHelperText>
```

Comment:
These 7 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1266-20230413-122810-fbb171ea-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)